config.py

```diff
@@ -13,5 +13,5 @@ class Config:
     HOST = os.getenv("HOST", "0.0.0.0")
     PORT = os.getenv("PORT", 5051)
 
-    DATABASE_SCHEMA = os.getenv("DATABASE_SCHEMA", "securemessage")
+    DATABASE_SCHEMA = os.getenv("DATABASE_SCHEMA", "securemessages")
```

Comment:
Is this change correct? At some point the schema name was deliberately changed from `securemessages` to `securemessage` to correct what was felt should be a singular and not a plural name. This was to align with the services names, as per the other applications.

This was done _after_ the app had been initially deployed to preproduction and therefore there were two schemas in the new `securemessage` database. The redundant one is `securemessages` and the correct one should be `securemessage`. Both the Cloud SQL database _and_ the schemaname should be singular `securemessage`. When deployed to production, we don't want the `securemessages` schema to be created.